core/src/main/res/values/strings.xml

```diff
@@ -94,74 +94,74 @@
     <string name="reset">Reset</string>
     <string name="url_label">URL</string>
     <string name="support_funding_label">Support</string>
-    <string name="support_podcast">Support this Podcast</string>
+    <string name="support_podcast">Support this Creator</string>
     <string name="error_label">Error</string>
     <string name="error_msg_prefix">An error occurred:</string>
     <string name="refresh_label">Refresh</string>
-    <string name="external_storage_error_msg">No external storage is available. Please make sure that external storage is mounted so that the app can work properly.</string>
+    <string name="external_storage_error_msg">Please mount external storage so that the app access it first.</string>
     <string name="chapters_label">Chapters</string>
     <string name="chapter_duration">Duration: %1$s</string>
     <string name="description_label">Description</string>
     <string name="shownotes_label">Shownotes</string>
-    <string name="shownotes_contentdescription">swipe up to read shownotes</string>
+    <string name="shownotes_contentdescription">Swipe ↑ to read shownotes</string>
     <string name="episodes_suffix">\u0020episodes</string>
     <string name="processing_label">Processing</string>
     <string name="close_label">Close</string>
     <string name="retry_label">Retry</string>
-    <string name="auto_download_label">Include in auto downloads</string>
-    <string name="auto_delete_label">Auto Delete Episode</string>
+    <string name="auto_download_label">Include in auto-downloads</string>
+    <string name="auto_delete_label">Auto-delete Episode</string>
     <string name="feed_volume_reduction">Volume Reduction</string>
-    <string name="feed_volume_reduction_summary">Turn down volume for episodes of this feed: %1$s</string>
+    <string name="feed_volume_reduction_summary">Reduce the volume for this episode feed: %1$s</string>
     <string name="feed_volume_reduction_off">Off</string>
     <string name="feed_volume_reduction_light">Light</string>
     <string name="feed_volume_reduction_heavy">Heavy</string>
-    <string name="parallel_downloads">%1$d parallel downloads</string>
-    <string name="feed_auto_download_global">Global default</string>
+    <string name="parallel_downloads">%1$d concurrent downloads</string>
+    <string name="feed_auto_download_global">For the whole app</string>
```

Comment:
```suggestion
    <string name="feed_auto_download_global">For the whole system</string>
```
?

Reply:
Just "global" is meaningless to me. Can you explain what action it is, maybe where to find it in the app?